wiki/Beatmap_Editor/Song_Setup/zh.md

```diff
@@ -160,7 +160,7 @@ Approach Rate通常缩写为"**AR**"。
 
 ### 物件颜色(Hitcircle/Slider Combos)
 
-这些选项可以直接规定某一串combo的颜色。对于一个谱面来说，combo的颜色是很重要的,一个铺面由很多滑条和圆圈组成，合适的颜色配置能提升谱面的视觉效果。 在页面上点击一个Combo将会打开你电脑上的取色板，然后你就可以选择对应的颜色了，记住这个颜色要适合和你的背景图像。你最多可以选择8个自定义颜色。点击“Remove Combo Color”会移除最后一个自定义颜色。
+这些选项可以直接规定某一串combo的颜色。对于一个谱面来说，combo的颜色是很重要的,一个铺面由很多滑条和圆圈组成，合适的颜色配置能提升谱面的视觉效果。 在页面上点击一个Combo将会打开你电脑上的取色板，然后你就可以选择对应的颜色了，记住这个颜色要适合和你的背景图像。你最多可以选择8个自定义颜色。点击“Remove Combo Colour”会移除最后一个自定义颜色。
```

Comment:
its that way is client i think so dont change

Reply:
I can't edit this file for some reason, github keeps breaking

Reply:
turns out google was auto translating the page -_-